### DISCUSSION TITLE
Tidak bisa pairing device

### DISCUSSION THREAD

**Discussion by HanustaviraGA** [kirimwa-id/forum#136] [Tanya Jawab]
![image](https://user-images.githubusercontent.com/68433367/190532774-7a4827d1-9aeb-4360-b136-e9fb94428a29.png)

Mohon pencerahannya mas @rioastamal , ketika hendak dipairing melalui perangkat Android, tidak terdeteksi dan tidak ada reaksi apa apa baik dari tampilan di layar komputer, maupun pada aplikasi WA di Android. Terima kasih

**Comment by rioastamal** [comment]
Mungkin waktu scan sudah experied, bisa dicoba lagi dengan lebih cepat.
